common/decisions/hfe_decision_plague.txt

```diff
@@ -60,6 +59,7 @@ hfe_decision_plague_planet_bombardment = {
 			fail_text = hfe_plague_clear_fail
 			has_planet_flag = hfe_plague_planet
 		}
+		owner = { allows_purge = yes }
```

Comment:
why you moved it

Reply:
I separated it to make multiple qualifiers that need to be met for the Decision.